[
  {
    "poster": {
      "poster_name": "Michael Bushe",
      "poster_handle": "michael125"
    },
    "time_stamp": "2020-10-07 07:06:09",
    "text": "I get a 404 on the batch read.  It’s not in the doc, is this how I use it?\nGET /crm/v3/objects/products/batch/read/&lt;hs_object_id&gt;"
  },
  {
    "poster": {
      "poster_name": "Michael Bushe",
      "poster_handle": "michael125"
    },
    "time_stamp": "2020-10-07 07:43:36",
    "text": "Ah, it’s not doc’ed but it’s in the Postman Collection. :slightly_smiling_face:"
  },
  {
    "poster": {
      "poster_name": "Matthew Hall",
      "poster_handle": "mhall"
    },
    "time_stamp": "2020-10-07 10:11:23",
    "text": "It is doc'ed too"
  },
  {
    "poster": {
      "poster_name": "Matthew Hall",
      "poster_handle": "mhall"
    },
    "time_stamp": "2020-10-07 10:14:43",
    "text": "I don't work for HS, so if it wasn't doc'ed I would not have known it existed"
  },
  {
    "poster": {
      "poster_name": "Matthew Hall",
      "poster_handle": "mhall"
    },
    "time_stamp": "2020-10-07 10:14:50",
    "text": "You have to scroll way down in the screen"
  },
  {
    "poster": {
      "poster_name": "Matthew Hall",
      "poster_handle": "mhall"
    },
    "time_stamp": "2020-10-07 10:28:09",
    "text": "This one is going to be hard for the community to debug, because it is a very new feature that requires an Enterprise license.\n\nBut my secret theory from reading <https://developers.HubSpot.com/docs/api/crm/crm-custom-objects?password_incorrect=true&amp;_ga=2.1785393.1255618442.1602021017-1278122576.1595997873> would be that maybe you need to do something with the `⇒ POST <https://api.hubapi.com/crm-object-schemas/v3/schemas/p99688696_car/associations?portalId=99688696>`,to configure the associations before that part will work and not error out"
  },
  {
    "poster": {
      "poster_name": "TaRan Wilson",
      "poster_handle": "taran.wilson"
    },
    "time_stamp": "2020-10-07 11:23:40",
    "text": "<@U012W4YEUTX> Thanks for the help. The code snippet did fix the error I was getting, but then I got a similar error for \"offset\" on the next line. I replaced the line with `parameter_dict['Offset']= response_dict.get('offset')`  and that fixed the new error. The script runs error-free now, but returns 0 records. Do you have any idea why it would do that? I have quite a few companies in my HubSpot list, so I'm missing something."
  },
  {
    "poster": {
      "poster_name": "TaRan Wilson",
      "poster_handle": "taran.wilson"
    },
    "time_stamp": "2020-10-07 11:24:29",
    "text": "You mentioned the code being buggy. It's the code, mostly, from HubSpot's documentation: <https://legacydocs.HubSpot.com/docs/methods/companies/get-all-companies>"
  },
  {
    "text": "Hi everyone :wave: hope you're all doing well. I need advice here :point_down::skin-tone-2:\n\nWe developed an app to connect Marketing Hub with Dynamics CRM and we're taking num_conversions_event as the only monitored event (to avoid big events cuota) we made some progress but after we update the same contact with a second form, for the same email but despite the fact that the num_conversion_events does increase (we consult it from the api), HubSpot does not detect this change and for this reason it does not send any modification event, if you query the HubSpot monitor can see that there is no sending information. \nCan something be done so that the numofevents property is detected by HubSpot when it is updated?\n\nThanks a lot for your help :relaxed:",
    "time_stamp": "2020-10-07 13:13:10",
    "reply_count": 3,
    "reply_users_count": 2,
    "replies": [
      {
        "time_stamp": "2020-10-07 19:12:23"
      },
      {
        "time_stamp": "2020-10-08 09:57:15"
      },
      {
        "time_stamp": "2020-10-08 11:02:37"
      }
    ]
  },
  {
    "poster": {
      "poster_name": "TaRan Wilson",
      "poster_handle": "taran.wilson"
    },
    "time_stamp": "2020-10-07 14:35:23",
    "text": "Alright, I figured out what it was. Apparently if you do not call out at least one property in the URL, it returns nothing. Thanks again for all of your help, Matthew."
  },
  {
    "poster": {
      "poster_name": "Matthew Hall",
      "poster_handle": "mhall"
    },
    "time_stamp": "2020-10-07 16:31:03",
    "text": "Yeah, that sounds like the kind of thing that could create some issues, but in general the code needs more error handling, too. I didn't spot it was their code as I always read the CURL statements and code my own Python with reams of logging code and bulletproofing, because I have to do that as my system has a lot of financial sector compliance rules, so I did not realize the default code had all those bugs until you reported it."
  },
  {
    "poster": {
      "poster_name": "Justin Goldby",
      "poster_handle": "justingoldby"
    },
    "time_stamp": "2020-10-07 17:48:25",
    "edited": {
      "time_stamp": "2020-10-07 17:49:19"
    },
    "text": "Hi Guys, when using postman or the test call widget in the API docs for this endpoint -  POST*`/crm/v3/objects/contacts/search`* , I get a successful 200 response and it shows the total number of records I have found, but the results array which is supposed to contain details about the indexes is empty - {\n\"total\": 3,\n\"results\": [],\n\"paging\": {\n\"next\": {\n\"after\": \"0\"\n},\n\"prev\": null\n}\n}\n\nAm I supposed to enable something in the test account connected to this app for it to show the properties of the records in the results array?  What am I missing here?",
    "reply_count": 3,
    "reply_users_count": 3,
    "replies": [
      {
        "time_stamp": "2020-10-07 18:49:51"
      },
      {
        "time_stamp": "2020-10-07 19:12:54"
      },
      {
        "time_stamp": "2020-10-08 18:42:49"
      }
    ]
  },
  {
    "poster": {
      "poster_name": "Justin Goldby",
      "poster_handle": "justingoldby"
    },
    "time_stamp": "2020-10-07 18:49:51",
    "text": "Disregard this, I found out you have to omit passing the \"limit\" param in the body or pass a number greater than 0 (the api docs are not very clear on what this param is doing!)"
  },
  {
    "poster": {
      "poster_name": "Matthew Hall",
      "poster_handle": "mhall"
    },
    "time_stamp": "2020-10-07 19:12:23",
    "text": "That might be a calculated property and not an actual data field being modified that you can monitor for deltas"
  },
  {
    "poster": {
      "poster_name": "Matthew Hall",
      "poster_handle": "mhall"
    },
    "time_stamp": "2020-10-07 19:12:54",
    "text": "Yeah, I was about to predict the `limit` had to be bogus"
  },
  {
    "poster": {
      "poster_name": "Hemasri Chappidi",
      "poster_handle": "hemasri.chappidi"
    },
    "time_stamp": "2020-10-08 00:27:36",
    "text": "Hello Everyone!\n\nI tried using this following code for pulling the data of *all contacts* and *all companies* from HubSpot. In the output file, I got all the rows of the dataset but not all the columns. I got to know this when I compared this data set with the manually exported data file. Could someone please suggest me what could have went wrong?\n\nfrom HubSpot3 import HubSpot3\n \nHubSpot = HubSpot3(api_key = &lt;API-Key&gt;\nall_contacts = HubSpot.contacts.get_all()\nall_companies = HubSpot.companies.get_all()",
    "reply_count": 1,
    "reply_users_count": 1,
    "replies": [
      {
        "time_stamp": "2020-10-08 11:02:21"
      }
    ]
  }
]